ddmrp/models/stock_move.py

```diff
@@ -21,3 +21,48 @@ def _merge_moves_fields(self):
         res = super(StockMove, self)._merge_moves_fields()
         res["buffer_ids"] = [(4, m.id) for m in self.mapped("buffer_ids")]
         return res
+
+    def write(self, vals):
+        res = super(StockMove, self).write(vals)
+        if "state" in vals and self.env.company.ddmrp_auto_update_nfp:
```

Comment:
Not all states update should trigger a recomputation

Reply:
which ones don't? I think any status change can potentially affect NFP (note that reservations reduce on-hand from buffer PoV - as qty available not reserved is what is considered on-hand)

Reply:
@LoisRForgeFlow done is not affecting the nfp